src/app/(views)/managed-courses/[courseId]/modules/[moduleId]/edit/page.tsx

```diff
@@ -75,38 +80,110 @@ export default function EditModule() {
   }
 
   return (
-    <CForm onSubmit={handleSubmit(onSubmit)}>
-      <CInputGroup>
-        <CFormLabel htmlFor="title">Title</CFormLabel>
-        <CFormInput id="title" {...register('title', { required: true })} />
-        {errors.title && <CFormText className="text-danger">This field is required</CFormText>}
-      </CInputGroup>
-      <CInputGroup>
-        <CFormLabel htmlFor="description">Description</CFormLabel>
-        <CFormInput id="description" {...register('description', { required: true })} />
-        {errors.description && (
-          <CFormText className="text-danger">This field is required</CFormText>
-        )}
-      </CInputGroup>
-
-      <CInputGroup>
-        <CFormLabel htmlFor="type">Type</CFormLabel>
-        <CFormSelect id="type" {...register('type', { required: true })}>
-          {typeOptions.map((option) => (
-            <option key={option.value} value={option.value}>
-              {option.label}
-            </option>
-          ))}
-        </CFormSelect>
-        {errors.type && <CFormText className="text-danger">This field is required</CFormText>}
-      </CInputGroup>
-
-      <ModuleContentInput type={watch('type')} value={watch('content')} setValue={setValue} />
-
-      <CButton type="submit" color="primary" disabled={updatingModule}>
-        {updatingModule ? <CSpinner size="sm" /> : 'Save'}
-      </CButton>
-    </CForm>
+    <>
```

Comment:
Do we need this commented code? if yes, put a comment above explaining why this is still here.